app/livechat/server/lib/Helper.js

```diff
@@ -535,3 +535,11 @@ export const updateDepartmentAgents = (departmentId, agents, departmentEnabled)
 
 	return true;
 };
+
+export const validateEmail = (email) => {
+	const emailValidation = /^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
```

Comment:
Please move this lite to outside the function, no need to assign it on call, it's a global constant